background/get-addon-settings.js

```diff
@@ -7,49 +7,33 @@ chrome.storage.sync.get(["addonSettings", "addonsEnabled"], ({ addonSettings = {
       const settings = addonSettings[addonId] || {};
       let madeChangesToAddon = false;
       if (manifest.settings) {
-        if (addonId === "editor-dark-mode") {
```

Comment:
maybe we should keep old addon transition code?

Reply:
why, its from like 5 versions